src/Exports.jl

```diff
@@ -406,6 +406,7 @@ export degree
 export degrevlex
 export dehomogenization
 export dehomogenize
```

Comment:
Remove the previous export?

Reply:
The name is also used by others?  @lkastner  is that part of your interface?
```
  [1] dehomogenize(vec::AbstractVector) in Oscar at /home/simon/.julia/dev/Oscar/src/PolyhedralGeometry/helpers.jl:178
  [2] dehomogenize(mat::AbstractMatrix) in Oscar at /home/simon/.julia/dev/Oscar/src/PolyhedralGeometry/helpers.jl:179
```